meson_options.txt

```diff
@@ -0,0 +1,78 @@
+# Use this for some compilers that doesn't C11 standard.
+# It is not needed for some non-compliant compilers such as
+# recent gcc and clang.
+
+option('flatcc_portable', type : 'boolean', value : false)
```

Comment:
Although common in CMake you don't need to prefix the options with `flatcc`; if needed in Meson you can add the library name as a prefix to things anyway, so `-Dflatcc:portable` would look for the flatcc library and set the portable flag.

Of course if you are just compiling from flatcc you don't need to specify this, and could just go with the shorter `-Dportable=` command line argument

Reply:
Makes sense, I've removed the `flatcc_` prefix for all the options as you suggested. I'll push up a new version of this PR in the near future once I have implemented as much of the feedback as possible.